section3/reflection.md

```diff
@@ -1,17 +1,52 @@
 ## Section 3 Reflection
 
 1. How different did your workflow feel this week, considering we asked you to follow the Pomodoro technique?
+* There were a couple times where it helped me step away and come back with more focus.
+* Other times, I was unable to use the technique because I would get annoyed by the idea of getting distracted while in the middle of a project.
+* I am not sure this technique is for me, as I will be focused for a long period of time, but I can get distracted or pulled away for more than 3 minutes too easily - cooking, dog, cleaning etc.
 
 2. Regarding the work you did around setting intentions in Step 1 of the Pomodoro technique - how did that go? Were you surprised by anything (did you find yourself way more focused than you realized, more distracted that you thought you'd be, estimating times accurately or totally off, etc)?
+* It went ok. I had to stop using pom because i felt it was interrupting my progress or focus.
+* Time flew. 9:30am turned to 1pm turned to 3:30pm by this point in this section. It usually does when I get focused.
 
 3. What is a Hash, and how is it different from an Array?
+* I want to say it a fancy array but that will make someone mad
+* Instead of only using index [0,1,2...] you can chose the index name as whatever you want
+* curly brackets "{}" are used instead of brackets
+* (apologies if this is incorrect terminology for {})
 
 4. In the space below, create a Hash stored to a variable named `pet_store`.  This hash should hold an inventory of items and the number of that item that you might find at a pet store.
 
+```ruby
+pet_store = {
+  dog_food: 14,
+  cages: 3,
+  squeak_toys: 22,
+  chew_toys: 32}
+```
+
 5. Given the following `states = {"CO" => "Colorado", "IA" => "Iowa", "OK" => "Oklahoma"}`, how would you access the value `"Iowa"`?
 
+```ruby
+states.values[1]
```

Comment:
Can you figure out how to access this with a single method call? (To clarify, `.values` is a Hash method that returns an Array, and then you use the bracket notation (an Array method) to access the value at index 1.

What Hash method returns the value associated with a given key? What does it return if that key doesn't exist in the Hash?  (this is working a bit ahead in Mod 1, but I think you can handle it!)

You used it in the hash exercise from learn ruby the hard way, but try to use [ruby-doc.org](https://ruby-doc.org/core-2.7.2/Hash.html) to find the answer instead.